server/http.go

```diff
@@ -160,3 +168,24 @@ func (s *HTTPd) RegisterShutdownHandler(callback func()) {
 		api.Methods(r.Methods...).Path(r.Pattern).Handler(r.HandlerFunc)
 	}
 }
+
+// ListenAndServe opens a listening socket (TLS if necessary) and starts HTTP server
+func (s *HTTPd) ListenAndServe() error {
```

Comment:
Pass the certificate here? No need for a struct member.

Reply:
Plus, this needs documentation in evcc.dist.yaml.

Reply:
Good suggestion.